src/decays/specializations/Ah/decay_Ah_to_gg.inc

```diff
@@ -18,9 +18,17 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, Gluon, Gluon>(
 
    double result = flux * color_fact * ps * ps_symmetry * amp.square();
 
+   const int Nf = number_of_active_flavours(qedqcd, mAh);
+   if (Nf < 5) {
+      problems.add_warning(
+         create_process_string<PseudoscalarHiggs,Gluon,Gluon>(in_idx, out1_idx, out2_idx)
+         + ": Cannot determine the number of active quark flavours. Disabling higher-order corrections."
```

Comment:
Is this error message correct? From looking at the if-statement it seems that the error is that `mAh` is actually too small.